specification/azsadmin/resource-manager/network/Microsoft.Network.Admin/preview/2015-06-15/examples/Locations/List.json

```diff
@@ -0,0 +1,15 @@
+{
+  "parameters": {
+    "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "location": "westus",
+    "api-version": "2015-06-15"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [],
```

Comment:
You don't have to, but to me an empty list does not provide much insight as an example...

Reply:
I took the operation example as a the template for this example. At this time, the location operation isn't really supported OnPrem. This operation is here to keep consistency with public azure.